lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -479,7 +479,7 @@ def pick(x):
             return x
 
     for grant in grants:
-        res = re.match("""GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\5)? ?(.*)""", grant[0])
+        res = re.match("""GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\6)? ?(.*)""", grant[0])
```

Comment:
I will leave it to smarter people than I to tell me if this is right.

Reply:
https://github.com/ansible/ansible/commit/e73e9ecbc338f38105e37e1b14dc1cc389ab3657

Reply:
Shouldn't this be
```suggestion
        res = re.match("""GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\\6)? ?(.*)""", grant[0])
```
?

Reply:
(Also, how about using raw strings?)